lib/deaf_grandma.rb

```diff
@@ -12,16 +12,26 @@ def run!
     print_welcome
 
     loop do
-      user_input = get_user_input
-      p speak(user_input)
+       user_input = get_user_input
```

Comment:
Look into your tab spacing. In ruby-land, 2 spaces are typically used for indentation.